sormas-api/src/main/java/de/symeda/sormas/api/event/EventDto.java

```diff
@@ -120,11 +122,11 @@ public class EventDto extends SormasToSormasShareableDto {
 	public static final String OTHER_DELETION_REASON = "otherDeletionReason";
 
 	private EventReferenceDto superordinateEvent;
-
-	@Required
+	@NotNull(message = Validations.validEventStatus)
```

Comment:
The `@Required` annotation is currently used in EntityColumn.getRequired. It's ok to replace it here, but then we also have to extend this method to check for both annotations.

I'd also suggest to add a comment to the Required annotation to use NotNull instead and maybe even deprecate it.

Reply:
I deprecated `@Required`. I wanted to do this for two years now... :smile: